From 9087d76cb76018b315306f6046b930b89d48fdb0 Mon Sep 17 00:00:00 2001
From: Chris Morgan <macromorgan@hotmail.com>
Date: Wed, 30 Oct 2024 17:54:04 -0500
Subject: [PATCH 03/10] arm64: dts: rockchip: Add vcc for vibrator node on
 RGxx3

Define the vcc supply for the vibrator on the Anbernic RGxx3 series
devices. This supply value was discovered by enabling the pwm channel
to full duty and then slightly raising and lowering the voltages of
various regulators from within the bootloader and listening for
harmonic changes from the vibrator.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
index 233eade30f21..f4c753f5d335 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
@@ -198,6 +198,7 @@ vibrator: pwm-vibrator {
 		compatible = "pwm-vibrator";
 		pwm-names = "enable";
 		pwms = <&pwm5 0 1000000000 0>;
+		vcc-supply = <&vcc_3v3>;
 	};
 };
 
-- 
2.43.0

